springboard.yaml

```diff
@@ -43,7 +43,7 @@ models:
         type: string
 repositories:
```

Comment:
well, we'll be deploying this to both QA and Prod so in essence you need different configs